jekyll/_api/v1-reference.md

```diff
@@ -4,72 +4,163 @@ title: CircleCI API v1.1 Reference
 categories: [reference]
 description: Using the CircleCI API
 ---
+## API Overview
 
-The CircleCI API is a fully-featured JSON API that allows you to access all information and trigger all actions in CircleCI. **Note:** Access to billing functions is only available from the CircleCI application.
+The CircleCI API is a full-featured RESTful API that allows you to access all information and trigger all actions in CircleCI. RESTful APIs enable you to call individual API endpoints to perform the following actions:
+<dl>
+  <dt>
+    GET - retrieve specific information, which may include arrays and sets of data and information.
+  </dt>
+  <dt>
+    POST - create/add a new API element.
+  </dt> 
+  <dt>
+    PUT - update an existing API element in the API server.
+  </dt>
+  <dt>
+    DELETE - remove/delete an API element in the API server.
+  </dt>
+  <dt>
+</dl>
+
+Although RESTful APIs include these 4 HTTP verbs, the CircleCI API does not currently use the PUT verb.
+
+### API Syntax
+
+When making an API request, make sure you follow standard REST API syntax and formatting. Adhering to proper REST API syntax ensures that the API server can properly process your request and return a JSON response. To make a request to the CircleCI API, use the format shown below:
+
+"https://circleci.com/api/v1.1"
+
+Where:
+
+<dl>
+  <dt>
+  https://circleci.com - the resource URL for the API being called.
+  </dt>
+  <dt>
+  api - the class being called.
+  </dt>
+  <dt>
+  v1.1 - the API version.
+  </dt>
+</dl>
+
+**Note:** Access to billing functions is only available from the CircleCI application.
+
+### Authentication
+
+The CircleCI API utilizes token-based authentication to manage access to the API server and validate that a user has permission to make API requests. Before you can make an API request, you must first add an API token and then verify that you are authenticated by the API server to make requests. The process to add an API token and have the API server authenticate you is described in the sections below.
```

Comment:
Might be worth noting that we also accept cookies, so if you're logged in you can hit these endpoints from the browser.

Reply:
:+1: